src/BulletSoftBody/btSoftBodyHelpers.h

```diff
@@ -17,7 +17,8 @@ subject to the following restrictions:
 #define BT_SOFT_BODY_HELPERS_H
 
 #include "btSoftBody.h"
-
+#include <fstream>
```

Comment:
not a fan of STL in a header file, wonder if we can only have the char* version of file, and implement all inside the c++ to avoid exposing STL in header.

Reply:
It'd be great if we could leave this job for the next patch.